views/aboutdiv.hjs

```diff
@@ -0,0 +1,31 @@
+<h1>About the R-hub builder</h1>
+<p>The R-hub builder offers free R CMD Check as a service on different platforms. It is a project by Gábor Csárdi, supported by the R Consortium.</p>
+<h2>Why use the R-hub builder?</h2>
+<p>The builder allows you to check your R package on several platforms, and R versions.</p>
+<p>Moreover, as a side-effect it also allows you to <em>build</em> your R package on several platforms, and R versions.</p>
```

Comment:
We should mention that "buiding" means building binaries, and building binaries for all  (source) dependencies as well. Maybe?

Reply:
the more details the better!